### DISCUSSION TITLE
bitte keine (technischen) Anglizismen

### DISCUSSION THREAD

**Discussion by Zeppelin-PV** [solectrus/solectrus#3454] [General]
![grafik](https://github.com/user-attachments/assets/83501cf7-6310-49f9-b3fb-8494e4c41e2d)

![grafik](https://github.com/user-attachments/assets/ab385a15-cea2-4c75-b3f0-13216b3ff509)

**Comment by ledermann** [comment]
Danke für den Hinweis! Hier handelt es sich um einen unbeabsichtigten Darstellungsfehler in `develop`, der sich gestern durch #3349 eingeschlichen hat: 

Wenn der Sensor `CAR_BATTERY_SOC` **nicht** vorhanden ist (also wenn man kein E-Auto hat und somit auch keinen Ladestand des E-Autos), dann wird der Ladestand des Haus-Speichers falsch bezeichnet ("SOC" statt korrekt "LADESTAND").

Ich habe das gerade korrigiert.
